File: CS/GettingStarted2/Models/NwindDbContext.cs
namespace GettingStarted2.Models {
    using System;
    using System.Data.Entity;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Linq;

    public partial class NwindDbContext : DbContext {
        public NwindDbContext()
            : base("name=NwindDbContext") {
        }

        public virtual DbSet<Category> Categories { get; set; }
        public virtual DbSet<Product> Products { get; set; }

        protected override void OnModelCreating(DbModelBuilder modelBuilder) {
            modelBuilder.Entity<Product>()
                .Property(e => e.UnitPrice)
                .HasPrecision(10, 4);

            modelBuilder.Entity<Product>()
                .Property(e => e.EAN13)
                .IsUnicode(false);
        }
    }
}

File: VB/GettingStarted2/Models/NwindDbContext.vb
Imports System
Imports System.Data.Entity
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.Linq

Partial Public Class NwindDbContext
    Inherits DbContext

    Public Sub New()
        MyBase.New("name=NwindDbContext")
    End Sub

    Public Overridable Property Categories As DbSet(Of Category)
    Public Overridable Property Products As DbSet(Of Product)

    Protected Overrides Sub OnModelCreating(ByVal modelBuilder As DbModelBuilder)
        modelBuilder.Entity(Of Product)() _
            .Property(Function(e) e.UnitPrice) _
            .HasPrecision(10, 4)

        modelBuilder.Entity(Of Product)() _
            .Property(Function(e) e.EAN13) _
            .IsUnicode(False)
    End Sub
End Class
